src/CTA.WebForms2Blazor/FileConverters/ViewFileConverter.cs

```diff
@@ -80,8 +81,10 @@ public override async Task<IEnumerable<FileInformation>> MigrateFileAsync()
 
             // View file converters will return razor file contents with
             // only view layer, code behind will be created in another file
```

Comment:
Please handle the warning on this line "This async method lacks 'await' operators and will run synchronously." I understand this is overriding a method and we cannot change the signature. Is there a DocumentNode.WriteToAsync maybe we can use to await? Perhaps the ConvertEmbeddedCode could be async as it appears to be a fairly intensive method computation wise. If all the above fail we can just Task.FromResult() on the return value of this function to suppress warning. There are several other warnings such as this as well.

Reply:
Yeah I think there are a lot of methods right now that have that same warning, I'll discuss with Emery and decide what we want to do about it

Reply:
I agree with Jonathan and second everything he said. `Task.FromResult()` seems to be the common way to handle the situation we're in (have an async interface where some implementations are synchronous): https://stackoverflow.com/a/19696462